### PR TITLE
Add pipelines common file and move to eng/pipelines

### PR DESCRIPTION
**Pull request by ViktorHofer**

### CODE REVIEW

File: eng/pipelines/azure-pipelines.yml [hunk 1]
```diff
@@ -88,4 +87,8 @@ stages:
             displayName: Unix Build / Publish
 
 - ${{ if eq(variables._RunAsInternal, True) }}:
-  - template: eng\common\templates\post-build\post-build.yml
\ No newline at end of file
+  - template: eng\common\templates\post-build\post-build.yml
```
Comment (safern) [inline eng/pipelines/azure-pipelines.yml]: What feed is this pipeline going to publish to? 

I think we shouldn't be publishing to dotnet-core, should we?

Comment (ViktorHofer) [reply eng/pipelines/azure-pipelines.yml]: This publishes to the channel's corresponding feed, dotnet5-transport.

Comment (safern) [reply eng/pipelines/azure-pipelines.yml]: Cool. Good to know it doesn’t publish to dotnet-core.

File: src/System.ComponentModel.TypeConverter.TestData/System.ComponentModel.TypeConverter.TestData.csproj [hunk 2]
```diff
@@ -1,7 +1,2 @@
 <Project Sdk="Microsoft.NET.Sdk">
-
-  <PropertyGroup>
-    <PackageVersion>1.0.4</PackageVersion>
```
Comment (safern) [inline src/System.ComponentModel.TypeConverter.TestData/System.ComponentModel.TypeConverter.TestData.csproj]: What’s the package version now going to be?

Comment (ViktorHofer) [reply src/System.ComponentModel.TypeConverter.TestData/System.ComponentModel.TypeConverter.TestData.csproj]: it's going to 5.0.0-... something. Either alpha or beta, whatever the default is right now. Will figure this out when I do the internal builds.

**Comment by ViktorHofer** [comment]
Merging even though signing isn't fixed yet.

**Pull request merged**